src/expr/src/relation/mod.rs

```diff
@@ -1429,9 +1429,9 @@ impl MirRelationExpr {
     }
 
     /// True iff the expression contains a `NullaryFunc::MzLogicalTimestamp`.
-    pub fn contains_temporal(&mut self) -> bool {
```

Comment:
Awww I've wanted to do this for a while but didn't want to muck with the visit fns. Thanks.

Should probably ask a compute person to eyeball this too.